.changeset/breezy-tigers-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-dhis2': major
+---
+
+Update trackedInstances to use new v42 implementation
```

Comment:
We can be a bit more helpful to users with these release notes.

What changes? What breaks? What do users need to know?

The headline feature is probably: `Add support for new Tracker and remove support for API versions <36` (or whatever)

And by the way, somewhere in this PR, if the API version is too low - anything before the tracker was released - we should put out a warning (sort of the opposite of what we're doing in the other one)